src/js/views/scores.js

```diff
@@ -20,6 +20,18 @@ define('views/scores',[
                 $scope.rev = (String($scope.sort) === String(col)) ? !$scope.rev : !!defaultSort;
                 $scope.sort = col;
             };
+
+            $scope.sortIcon = function(col){
```

Comment:
This code looks radically different from the one in ranking.js, but seems to basically do the same thing. Can we refactor it out into a single function somewhere?

Reply:
We can, but currently I'm not sure where. What I'll do in the meanwhile is change ranking.js's sortIcon() to be a similar as possible to scores.js's, and if and when we decide on a place that will have the global sortIcon(), it'll be easy to refactor those two places to use it.

Reply:
Ok, nice.